### ISSUE TITLE
Multiline arrow functions inside function calls cause a crash

### ISSUE THREAD

**Issue by alangpierce** [decaffeinate/decaffeinate#261]
Here's a reduced example:
```
a(() ->
  0)
```

It gives the error `Unexpected token (2:11)` in the decaffeinate repl:
http://decaffeinate.github.io/decaffeinate/repl/#?evaluate=true&code=a(()%20-%3E%0A%20%200)

**Comment by madskonradsen** [comment]
Looks like the patcher is a tad confused... `}` or `)` is emitted in the wrong place

```
NormalizeStage a(() ->
  0)
MainStage a(() ->
  0)
FunctionPatcher INSERT RIGHT 2 "function" BEFORE "() ->\n  "
FunctionPatcher OVERWRITE [5, 7) "->" → "{"
FunctionPatcher INSERT RIGHT 10 "return " BEFORE "0)"
BlockPatcher INSERT RIGHT 11 ";" BEFORE ")"
BlockPatcher INSERT RIGHT 12 "\n}" BEFORE ""
BlockPatcher Moving `innerEnd` from 11 to 12
BlockPatcher Moving `outerEnd` from 11 to 12
FunctionPatcher Moving `innerEnd` from 11 to 12
FunctionPatcher Moving `outerEnd` from 11 to 12
BlockPatcher INSERT RIGHT 12 ";" BEFORE ""
AddVariableDeclarationsStage a(function() {
  return 0;)
};
```

**Comment by alangpierce** [comment]
Thanks!

**Comment by eventualbuddha** [comment]
Fixed in v2.13.2.
